changelog/changed-use-write-for-ram-flash.md

```diff
@@ -0,0 +1,4 @@
+Use `write` instead of `write_8` for loading images to RAM. This can be a performance
```

Comment:
I don't think this deserves a changelog entry, let alone a short story with it.

Reply:
Removed it